packages/react-slider/Spec.md

```diff
@@ -1,63 +1,230 @@
+
+### Visual behavior props
+
+| Name              | <img src="https://img.shields.io/badge/Used%20in-v0-orange" alt="drawing" width="200"/> | <img src="https://img.shields.io/badge/Used%20in-v8-blue" alt="drawing" width="200"/> | Description                                                                                                   |
+| ----------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- |
+| label             | x                                                                                       | &check;                                                                               | The description label of the **Slider**.                                                                      |
+| valueLabel        | x                                                                                       | &check;                                                                               | The current value or unique format to be shown for the **Slider's** `value label`.                            |
+| disabled          | &check;                                                                                 | &check;                                                                               | Whether to render the **Slider** as disabled. @defaultvalue `false` (render enabled)                          |
+| vertical          | &check;                                                                                 | &check;                                                                               | Whether to render the **Slider** vertically. @default `false` (render horizontally)                           |
+| marks             | x                                                                                       | x                                                                                     | Whether the **Slider** will have marks to visibly display its steps. @default `false` (renders without marks) |
+| fluid             | x                                                                                       | &check;                                                                               | A **Slider** can take the width of its container. @default `false` (width does not fill the container)        |
+| disabledFocusable | X                                                                                       | X                                                                                     | A **Slider** can be disabled and focusable at the same time.                                                  |
```

Comment:
This seems weird to me -- `disabledFocusable` makes sense for a button or menuitem, but why is this an option for a Slider?

Reply:
The accessibility design mentioned that the `disabled` Slider should `allow` focus but be readonly to assist screen readers in determining if an element was disabled. The `disabledFocusable` event could be added to follow similar patterns in other converged components. Out of curiosity, how would you suggest handling the `disabled` behavior of the Slider? Should it allow focus in the first place?